### ISSUE TITLE
Emails from admin system for login taking minutes to deliver

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/backend#404]
I know we've been over time many times, and each time we're getting better. But! During a user test it took a few minutes for the login email to be delivered, but an email from the request system was instant ( I sent that to test out manual vs automatic sending). During this time, I was able to send & receive a message from the login system. 

I know we're having trouble replicating this internally so I grabbed some data for it. The email headers for one of the admin emails that was slow & a fast request system email is below. 

[original_msg (1).txt](https://github.com/OAButton/backend/files/608782/original_msg.1.txt)
[original_msg.txt](https://github.com/OAButton/backend/files/608783/original_msg.txt)

There are also some potentially helpful logs in Mailgun itself. 

Putting these into https://toolbox.googleapps.com/apps/messageheader/analyzeheader shows the source of the delay. However, that's where my understanding of all this ends. I realise this is a discussion we've had many times, but perhaps data like this can be helpful?

**Comment by meganwaps** [comment]
It usually seems to take the longest when I attempt to log in. In the past it has taken more than 30 minutes for an email to arrive in my inbox. However I just tried using Chrome and the email was delivered immediately. I'm going to attempt to actually get data so I'll be logging in/out a lot today. I'll try different browsers, and change settings such as developer mode, ad blocker, etc to get some data. I'll start a spreadsheet in our Drive:
https://docs.google.com/spreadsheets/d/1T33A6uLXS6ERIlkfa5HABFKWgw-ZqwThREHmFHCytZk/edit?usp=sharing

It's very basic but I'd love advice on other variable to test and check. So far I haven't had any issues today. It is an incredible improvement from about 3 weeks ago.

**Comment by meganwaps** [comment]
Update: It has been several minutes since I requested a log in code via Firefox and I still have not received an email. @JosephMcArthur @markmacgillivray 

I also noticed something interesting, a difference between FF and the other browsers I've used today. 

The toolbar on safari and chrome look like this
![screen shot 2016-11-28 at 1 20 23 pm](https://cloud.githubusercontent.com/assets/5968703/20686423/7b332d14-b56d-11e6-95b4-96ea7ccad636.png)

It says 'My Account' but then asks me to request an email code, no problem. 

But then on FF, 
![screen shot 2016-11-28 at 1 21 26 pm](https://cloud.githubusercontent.com/assets/5968703/20686471/a485bc2c-b56d-11e6-832e-a1e4d5978531.png)

It says 'Login' instead of 'My Account', even though I've logged in numerous times on FF and it should have my credentials saved. It does not. I'm not sure if this is an error on my end or not. However it has been 6 minutes and I still have not received an email after trying to use FF.
